core/src/api/files.rs

```diff
@@ -43,11 +47,49 @@ pub(crate) fn mount() -> AlphaRouter<Ctx> {
 						.db
 						.object()
 						.find_unique(object::id::equals(args.id))
-						.include(object::include!({ file_paths media_data }))
+						.include(object::include!({ file_paths }))
 						.exec()
 						.await?)
 				})
 		})
+		.procedure("getMediaData", {
+			#[derive(Type, Deserialize)]
+			pub enum MediaDataType {
+				Image,
+				Video,
+				Audio,
+			}
+
+			#[derive(Type, Deserialize)]
+			pub struct GetMediaDataArgs {
+				pub id: i32,
+				pub md_type: MediaDataType,
+			}
+
+			R.with2(library())
+				.query(|(_, library), args: GetMediaDataArgs| async move {
+					let i = library
+						.db
+						.object()
+						.find_unique(object::id::equals(args.id))
+						.include(object::include!({ media_data }))
```

Comment:
`select` would be nicer here
```suggestion
						.select(object::select!({ id kind media_data }))
```

Reply:
oh good catch, bit redundant selecting media data when we're querying media data